docs(multizone): update instruction on multizone (#1354)
kumahq/kuma-website: app/_src/production/cp-deployment/multi-zone.md

1.  Set the `controlPlane.mode` value to `global` in the chart (`values.yaml`), then install. On the command line, run:

    ```sh

>     helm install {{ site.mesh_helm_install_name }} --create-namespace --namespace {{site.mesh_namespace}} --set {{set_flag_values_prefix}}controlPlane.mode=global {{ site.mesh_helm_repo }}

Reviewer: should it be `site.set_flag_values_prefix` also please prefix these with `mesh_`

Reviewer: Ha nevermind then. I didn't realise this alreayd existed!

{% tabs zone-control-planes useUrlFragment=false %}
{% tab zone-control-planes Kubernetes %}

1.  On each zone control plane, run:

    {% if_version gte:2.3.x %}
    ```sh
    kumactl install control-plane \
    --mode=zone \
    --zone=<zone name> \
    --ingress-enabled \
    --kds-global-address grpcs://<global-kds-address>:5685 \

>     --set controlPlane.tls.kdsZoneClient.skipVerify=true | kubectl apply -f -

Reviewer 2: This is not going to be ported well to KM where we need `kuma.` prefix. Same for all helm values.

Author: is it problem of this PR or all docs?

Btw. I thought we solved helm prefixes already...

Author: We do actually. Let me add this to the whole page `{{set_flag_values_prefix}}`

    ```
    {% endif_version %}
    {% if_version lte:2.2.x %}
    ```sh
    kumactl install control-plane \
    --mode=zone \
    --zone=<zone name> \
    --ingress-enabled \
    --kds-global-address grpcs://<global-kds-address>:5685 | kubectl apply -f -
    ```
    {% endif_version %}

    where `zone` is the same value for all zone control planes in the same zone.

    Add `--egress-enabled` to list of arguments if you want to deploy optional [Zone Egress](/docs/{{ page.version }}/production/cp-deployment/zoneegress/).

    {% if_version gte:2.3.x %}
    `--set controlPlane.tls.kdsZoneClient.skipVerify=true` is required because the default global control plane's certificate is self-signed.

>     It is recommended to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.

Reviewer 2: I guess maybe this to not use passive voice? :

```suggestion
    We recommend to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.
```

Author: It does not like We as much as passive voice

    It is recommended to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.
    {% endif_version %}

{% endtab %}
{% tab zone-control-planes Universal %}

1. On each zone control plane, run:

   {% if_version gte:2.3.x %}
   ```sh
   KUMA_MODE=zone \
   KUMA_MULTIZONE_ZONE_NAME=<zone-name> \
   KUMA_MULTIZONE_ZONE_KDS_TLS_SKIP_VERIFY=true \
   KUMA_MULTIZONE_ZONE_GLOBAL_ADDRESS=grpcs://<global-kds-address>:5685 \
   ./kuma-cp run
   ```
   {% endif_version %}
   {% if_version lte:2.2.x %}
   ```sh
   KUMA_MODE=zone \
   KUMA_MULTIZONE_ZONE_NAME=<zone-name> \
   KUMA_MULTIZONE_ZONE_GLOBAL_ADDRESS=grpcs://<global-kds-address>:5685 \
   ./kuma-cp run
   ```
   {% endif_version %}

   where `KUMA_MULTIZONE_ZONE_NAME` is the same value for all zone control planes in the same zone.

   {% if_version gte:2.3.x %}
   `KUMA_MULTIZONE_ZONE_KDS_TLS_SKIP_VERIFY` is required because the default global control plane's certificate is self-signed.

>    It is recommended to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.

Reviewer 2: ```suggestion
   It's recommended to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.
```
